new-dti-website-redesign/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,19 @@
+'use client';
+
+import IconWrapper from './IconWrapper';
+
+type FeatureCardProps = {
+  title: string;
+  body: string;
+  children: React.ReactNode;
+};
+
+export default function FeatureCard({ title, body, children }: FeatureCardProps) {
+  return (
+    <div className="p-[32px] max-w-[395px] border border-border-1 flex flex-col">
```

Comment:
Let's also try avoid using custom values when we have round numbers such as `32px`. 
```suggestion
    <div className="p-8 max-w-[395px] border border-border-1 flex flex-col">
```